packages/scope-manager/src/definition/CatchClauseDefinition.ts

```diff
@@ -6,8 +6,7 @@ import { DefinitionType } from './DefinitionType';
 class CatchClauseDefinition extends DefinitionBase<
   DefinitionType.CatchClause,
   TSESTree.CatchClause,
-  null,
-  TSESTree.BindingName
```

Comment:
I'll be honest when I say that keeping these for documentation is nice.
Can we invert this problem and remove the default from `DefinitionBase`?